src/backends/SFCCCors.js

```diff
@@ -0,0 +1,150 @@
+import { ProductSelectorError } from '../ProductSelectorError';
+import { get } from 'lodash';
+
+export class SFCCCors {
+  constructor(settings) {
+    this.settings = settings;
+    this.tokens = {};
+  }
+
+  tokenExpired(expires) {
+    const FIVE_MINS = 300000;
```

Comment:
Same as comment on line 46

Reply:
Addressed, now called `EXPIRY_BIAS` and has an explanation comment.